TF_Settings_and_Tooling_Unity/Assets/TouchFree/SettingsUI/Scripts/DiagnosticAPI.cs

```diff
@@ -413,7 +470,48 @@ class GetServerInfoResponse : DiagnosticApiResponse
         public GetServerInfoResponse() : base("GetServerInfo") { }
         public ServiceInfoPayload payload;
     }
-    
+
+    [Serializable]
+    class GetAllowImagesRequest : DiagnosticApiRequest
```

Comment:
Having not looked at the diagnostic API recently it took me a while to figure out why we have all these classes for request & response types. After a bit of digging I can see its to allow easy JSON serialisation, could we wrap them in a region and/or comment to make it clear for future readers?

Reply:
I've put these in a comment, it might even be worth putting them in a separate file